app/models/job.rb

```diff
@@ -10,6 +10,13 @@ class Job < ActiveRecord::Base
   belongs_to :organization
 
   scope :published, -> { where(state: :published).order(created_at: :desc) }
+  scope :search, (lambda do |query|
+    if query.present?
+      where(arel_table[:title].matches("%#{query}%").or(arel_table[:description].matches("%#{query}%")))
```

Comment:
You can simplify that line with good ol' sql
```
where("title LIKE :query OR description LIKE :query", query: "%#{query}%")
```
(note that I may have made a typo or a syntax mistake)